feat(tem): scale plan MTA-5622 (#4576)
scaleway/docs-content: pages/transactional-email/how-to/manage-blocklists.mdx

  - managed-services
---

> The Transactional Email (TEM) blocklist feature allows your to manage email addresses that generate errors, or “bounces”. Emails are automatically blocked when a mailbox does not exist, or is full, to prevent sending emails to a recipient address that is not reachable. You can also add emails to the blocklist manually if you subscribe to the [TEM Scale plan](/concepts#scale-plan) and have a Scaleway domain.

Reviewer: ```suggestion
The Transactional Email (TEM) blocklist feature allows you to manage email addresses that generate errors, or “bounces”. Emails are automatically blocked when a mailbox does not exist, or is full, to prevent sending emails to a recipient address that is not reachable. You can also add emails to the blocklist manually if you subscribe to the [TEM Scale plan](/concepts#scale-plan) and have a Scaleway domain.
```

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization. For blocklists your can apply the `TransactionalEmailBlocklistFullAccess` permission
- [Configured your domain(s) with Transactional Email](/transactional-email/how-to/configure-domain-with-transactional-email/)

## How to add an email address to the blocklist

> You can manually add addresses to your blocklist in the Scaleway console. Once added, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).

Author: ```suggestion
You can manually add addresses to your blocklist in the Scaleway console. Once added, the email remains blocked indefinitely until you [unblock it](# how-to-unblock-an-email-address) manually.
```

> You can manually add address to your blocklist in the Scaleway console. When you add an email to the blocklist, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).

Reviewer 2: ```suggestion
You can manually add addresses to your blocklist in the Scaleway console. When you add an email to the blocklist, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).
```

> You can manually add address to your blocklist in the Scaleway console. When you add an email to the blocklist, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).

Reviewer: ```suggestion
You can manually add an address to your blocklist in the Scaleway console. When you add an email to the blocklist, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).
```

> You can manually add addresses to your blocklist in the Scaleway console. When you add an email to the blocklist, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).

Author: ```suggestion
You can manually add addresses to your blocklist in the Scaleway console. Once added, it remains blocked indefinitely until you decide to [unblock it](#how-to-unblock-an-email-adress).
```

<Message type="important">
  Manually adding an email address to a domain's blocklist is only available for users who have a Scaleway domain and subscribe to the [TEM Scale plan](/concepts#scale-plan) for that domain's Project.
</Message>

1. Click **Transactional Email** in the **Managed Services** section of the [console](https://console.scaleway.com/) side menu. Your domains display.
2. Click the domain for which you want to manage the blocklist. An overview of your domain appears.
3. Click the **Blocklist** tab.
4. Click **Add to blocklist**. A pop-up appears
5. Enter the recepient emails you wish to prevent from receiving emails from your domain.
    <Message type="important">
      You can separate each email address by a comma to add multiple emails at once.
    </Message>

> 6. Optionally, add a reason for the block.

Reviewer 3: ```suggestion
6. (Optional) Provide a reason for blocking the address.
```
